SPECS/kernel/CVE-2023-23005.nopatch

```diff
@@ -0,0 +1,3 @@
+CVE-2023-23005 - The CVE has been disputed
+
+See https://nvd.nist.gov/vuln/detail/CVE-2023-23005 for more details.
```

Comment:
Looks like this cve is also not applicable, can you add those details as well even though the cve is disputed.